app/controllers/api/v1/greetings_controller.rb

```diff
@@ -1,12 +1,14 @@
+module Api
+  module V1
+    class GreetingsController < ApplicationController
```

Comment:
Kindly look into the following errors I got trying to run your project. These errors came after running the rails server and opening the port on the browser.

- This is from the console.
![image](https://user-images.githubusercontent.com/27709832/204908723-58934478-7b31-48a8-9b5c-975340d39839.png)

- This is from the page.
![image](https://user-images.githubusercontent.com/27709832/204908851-64711da9-2d80-4bf5-b78e-9fa1ad63fc45.png)

Reply:
> Kindly look into the following errors I got trying to run your project. These errors came after running the rails server and opening the port on the browser.
> 
> * This is from the console.
>   ![image](https://user-images.githubusercontent.com/27709832/204908723-58934478-7b31-48a8-9b5c-975340d39839.png)
> * This is from the page.
>   ![image](https://user-images.githubusercontent.com/27709832/204908851-64711da9-2d80-4bf5-b78e-9fa1ad63fc45.png)

- [ ] Kindly fix the issue addressed by the previous reviewer. I got the same issue while checking. :+1: